environments/utility/Dockerfile

```diff
@@ -29,7 +29,7 @@ RUN set -x && \
     usermod -L utility
 
 RUN set -x && \
-    gem install bundler --no-document -v 2.2.28 && \
+    gem install bundler --no-document -v 2.4.1 && \
```

Comment:
Looks like this doesn't work. Bundler 2.4 requires Ruby 2.6, but Ubuntu 18.04 has Ruby 2.5.

We could upgrade the utility image to Ubuntu 22.04, but then we'll also need to migrate from createrepo to createrepo_c, which is a bigger change. It's easiest to just stick with Bundler 2.2 for now.

Reply:
Makes sense, rollbacked it in 21dbe8d